apps/fhir/bluebutton/migrations/0002_archivedcrosswalk.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 2.2.24 on 2021-07-02 17:59
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('bluebutton', '0001_squashed_0006_auto_20210513_1812'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='ArchivedCrosswalk',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('_fhir_id', models.CharField(db_column='fhir_id', db_index=True, default=None, max_length=80)),
+                ('user_id_type', models.CharField(choices=[('H', 'HICN'), ('M', 'MBI')], default='H', max_length=1, verbose_name='Hash ID type last used for FHIR_ID lookup')),
+                ('_user_id_hash', models.CharField(db_column='user_id_hash', db_index=True, default=None, max_length=64, verbose_name='HASH of User HICN ID')),
+                ('_user_mbi_hash', models.CharField(db_column='user_mbi_hash', db_index=True, default=None, max_length=64, null=True, verbose_name='HASH of User MBI ID')),
+                ('archived_at', models.DateTimeField(auto_now_add=True)),
```

Comment:
Do you think we should go with the more generic database column names of: `updatedAt`, `createdAt`. Those can basically apply to any table and will give the same thing we are going for here

Reply:
> Do you think we should go with the more generic database column names of: `updatedAt`, `createdAt`. Those can basically apply to any table and will give the same thing we are going for here

Was keeping the naming similar to the other Archived* type models.

For example: 

https://github.com/CMSgov/bluebutton-web-server/blob/fe3ac84a83b37528d75c37674eddd6e389b85712/apps/dot_ext/models.py#L223
https://github.com/CMSgov/bluebutton-web-server/blob/7e33b3e557cf0159740515cdd9405b359fe92151/apps/authorization/models.py#L46

However, I am missing a copy of the `date_created` field from the Crosswalk instance.

I like the idea of us standardizing on `created_at` and `updated_at` for our models. There looks to be a mix of naming for those across our models.

I'll update the model shortly.

Reply:
Also don't think you need it but this link is super small and just shows the ez commands for those 2 fields:
https://timonweb.com/django/automatic-created-and-updated-datetime-fields-for-django-models/